### PR TITLE
[ts-sdk] Update legacy ts sdk dependencies

### PR DESCRIPTION
**Pull request by gregnazario**
### Description
Dependent on https://github.com/aptos-labs/aptos-core/pull/11267

### Test Plan
CI should show if tests work properly with the patches.  I haven't run it yet with a test dapp, that will be soon.

### CODE REVIEW

File: ecosystem/typescript/sdk/src/aptos_types/abi.ts [hunk 1]
```diff
@@ -30,7 +30,10 @@ export class ArgumentABI {
    * @param name
    * @param type_tag
    */
-  constructor(public readonly name: string, public readonly type_tag: TypeTag) {}
+  constructor(
+    public readonly name: string,
+    public readonly type_tag: TypeTag,
```
Comment (0xjinn) [inline ecosystem/typescript/sdk/src/aptos_types/abi.ts:35]: Are these coming from pnpm fmt? The comma after the last param seems weird lol

**Review by 0xjinn — APPROVED** [review]

File: ecosystem/typescript/sdk/src/aptos_types/abi.ts (hunk 1 above)
Comment (gregnazario) [reply ecosystem/typescript/sdk/src/aptos_types/abi.ts:35]: Yes, this comes from the formatter directly.  Commas are optional in most places, except JSON, where they're explicitly not allowed

**Comment by 0xmaayan** [comment]
Are you planning to publish a new version? Or just to shut down dependentbot messages?

If the former, thats fine. But we should set up a deadline where this package will not get anymore maintenance and updaes

**Review by 0xmaayan — APPROVED** [review]

**Comment by gregnazario** [comment]
> Are you planning to publish a new version? Or just to shut down dependentbot messages?
> 
> If the former, thats fine. But we should set up a deadline where this package will not get anymore maintenance and updaes

This is to handle dependabot messages, as well as just provide the last security fixes, but no new features.

I will deploy a new version.

**Pull request merged**